<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="cashier_table">
    <property name="geometry">
     <rect>
      <x>388</x>
      <y>1</y>
      <width>411</width>
      <height>581</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Item</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Count</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Price</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Total_price</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="cashier_total_lb">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>530</y>
      <width>81</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 200, 0);</string>
    </property>
    <property name="text">
     <string>Total : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_bt">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="check_bt">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Check</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalprice_lb">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>530</y>
      <width>81</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 200, 0);</string>
    </property>
    <property name="text">
     <string>0$</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="barcode_input">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>170</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Barcode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirm_bt">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
